usaspending_api/etl/tests/integration/test_spark_app.py

```diff
@@ -216,19 +233,27 @@ def test_spark_write_csv_app_run(spark: SparkSession, s3_unittest_data_bucket):
 
     df = spark.createDataFrame([Row(**data_row) for data_row in data])
     # NOTE! NOTE! NOTE! MinIO locally does not support a TRAILING SLASH after object (folder) name
-    df.write.option("header", True).csv(f"s3a://{s3_unittest_data_bucket}" f"/{CONFIG.DELTA_LAKE_S3_PATH}/write_to_s3")
+    csv_path = f"s3a://{s3_unittest_data_bucket}" f"/{CONFIG.DELTA_LAKE_S3_PATH}/write_to_s3"
+    if CONFIG.USE_FILESYSTEM_FOR_SPARK_DATA:
```

Comment:
I would think that this would always be true living inside of tests.

Reply:
@sethstoudenmier this should not be true when running in Travis. At that point, I think S3 can be used

Reply:
Forgot to update `travis.yml` to set `USE_FILESYSTEM_FOR_SPARK_DATA` to `False`